patterns.py

```diff
@@ -56,6 +56,14 @@
 
     'david':[11,19,20,21,22,23,34,33,32,31,30,42],
 
+    'flower':{
+            'pistil':[20,21,22,31,32,33],
+            #petals
+            'left':[24,25,26,27,28,29],
+            'top':[41,42,43,49,50,51],
+            'right':[16,17,18,35,36,37],
+            'bottom':[2,3,4,10,11,12]
+            },
```

Comment:
nitpick: add a line of space after this to stay consistent with the other entries